### PR TITLE
Finished Step 2: EDA

### PR DESCRIPTION
**Pull request by jackattackyang**
Hey Rayce,
I finished the preliminary EDA. Included in this PR: 
- 2 graphs (barplot and boxplot) in **data** folder
- R script that takes input and output file locations in **src** folder

Please have a look. Thanks!

### CODE REVIEW

**Comment by jackattackyang** [comment]
Finished Preliminary Analysis and Graph

File: src/pokemonEDA.R [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# bar graph displaying pokemon vs legendary pokemon numbers
+pokemon_count <- pokemon_data_tidy %>% 
+  ggplot(aes(Legendary, fill=Legendary)) + 
+  geom_bar(width=0.5) + labs(title="Pokemon Count", y="") + 
+  theme(legend.position="none")
+ggsave("EDA_pokemonCount.png", pokemon_count, path=output_file)
+
+# boxplot comparing pokemon stats
+pokemon_feature_comparison <- pokemon_data_tidy %>%
```
Comment (RayceRossum) [inline src/pokemonEDA.R]: This is an awesome analysis! Looks like we should have a pretty good fit for detecting legendaries simply based on their total stats. I wonder if there's anything we can do to handle the little bit of overlap.

File: src/pokemonAnalysis.py [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+    tree = DecisionTreeClassifier()
+    tree.fit(features, legendary)
+    feature_importances = tree.feature_importances_
+    importance_table = pd.DataFrame({"Feature": features.columns.values, "Importance": feature_importances})
+    importance_table = importance_table.query("Importance > 0.01")
+    importance_table = importance_table.sort_values(by="Importance", ascending=False)
+
+    plt.rcParams["figure.figsize"] = (15, 4)
+    plt.rcParams["xtick.labelsize"] = 10
+    importance_plot = sns.barplot(data=importance_table, x="Feature", y="Importance")
```
Comment (RayceRossum) [inline src/pokemonAnalysis.py]: We should save the output of this for our final report.

File: src/pokemonAnalysis.py [hunk 3]
```diff
@@ -1,11 +1,31 @@
...
 import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+%matplotlib inline
 from sklearn.tree import DecisionTreeClassifier, export_graphviz
 
-import pokemonData
+#import pokemonData
+pokemon_data = pd.read_csv("./data/Pokemon_clean.csv")
+pokemon_data = pd.get_dummies(pokemon_data, columns=["Type 1", "Type 2"])
```
[the remark this answers is not in the document]
Comment (jackattackyang) [reply src/pokemonAnalysis.py]: That makes sense. Sounds good.

**Pull request merged**